src/components/masthead/masthead.js

```diff
@@ -122,16 +116,22 @@ class Masthead extends React.Component {
   }
 }
 
-Masthead.propTypes = {
+PfMasthead.propTypes = {
```

Comment:
Do we want to be updating the prop types like this for PfMasthead instead of passing them in one by one like you had before?